PIL_Helper.py

```diff
@@ -199,8 +199,14 @@ def BuildPage(card_list, grid_width, grid_height, filename,
 def BlankImage(w, h, color=(255,255,255), image_type="RGBA"):
     return Image.new(image_type, (w, h), color=color)
 
-def LoadImage(filepath):
-    return Image.open(filepath)
+def LoadImage(filepath, fallback="blank.png"):
+    try:
+        return Image.open(filepath)
+    except Exception:
+        if fallback is not None:
```

Comment:
You can make this "if fallback:", so if it somehow gets a blank string, it will treat it like a None.